src/pages/login/__tests__/login.spec.ts

```diff
@@ -377,5 +385,15 @@ describe('LoginPage', () => {
       expect((tags[0].nativeElement as HTMLElement).textContent).toContain('not authorised to use this app');
     });
 
+    it('should show the correct div if user has an old version of the app', () => {
+      component.hasUserLoggedOut = false;
+      component.appInitError = AppConfigError.INVALID_APP_VERSION;
+      fixture.detectChanges();
+
+      const tags = fixture.debugElement.queryAll(By.css('h2'));
+      expect(tags.length).toBe(1);
+      expect((tags[0].nativeElement as HTMLElement).textContent).toContain('Unsupported application version');
+    });
+
```

Comment:
Would like to see an explicit test that the div is not there if app version is okay.

Reply:
will add now :)